packages/next/client/image.tsx

```diff
@@ -510,6 +510,17 @@ export default function Image({
           ) : null}
         </div>
       ) : null}
+      <noscript>
+        <img
+          {...rest}
+          {...imgAttributes}
+          src={src}
+          decoding="async"
+          style={{ ...imgStyle, visibility: 'visible' }}
+          className={className}
+          ref={thisEl}
```

Comment:
Is the ref still needed?

Reply:
you're right - the `ref` seems to only be used for the observer inside `useEffect`, which won't run when js is disabled, so it's not necessary